### PR TITLE
Doc/87 acceptance criteria and uc

### PR DESCRIPTION
**Pull request by alvarobernal2412**
# Description
I have created an "Information about the service" section where our MVP acceptance criteria and use cases are listed.

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:
**Acceptance criteria**
Captura de pantalla 2024-03-01 a las 13 39 03

### CODE REVIEW

File: docs/Información del servicio/Casos de uso.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Historial de versiones
+| Versión | Fecha      | Diferencia entre versiones                                                                                                                                                                                 |
+| ------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1.0     | 2024-03-01 | Versión inicial del documento. Incluye los casos de uso del Sprint 1, un identificador y una breve descripción del caso de uso.                                                                          |
+
+
+## Sprint 1
+
+| Identificador | Caso de Uso                                   | Descripción                                                                                                                |
+|---------------|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
```
Comment (AntonioRodriguezRuiz) [inline docs/Información del servicio/Casos de uso.mdx:10]: Could you add a context column (ACAT / CyC / BOTH)?

File: docs/Información del servicio/Casos de uso.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+| 1             | Acceso al Sistema                             | Los usuarios administradores pueden iniciar sesión en el sistema utilizando sus credenciales.                               |
+| 2             | Creación de Cuenta de Administrador          | Los administradores pueden crear nuevas cuentas de administración, definiendo niveles de acceso.                             |
+| 3             | Registro de Personas                          | Permite incluir información detallada sobre nuevas personas en el sistema.                                                   |
+| 4             | Consulta de Información de Personas           | Permite buscar y visualizar la información de las personas registradas en el sistema.                                        |
+| 5             | Actualización de Datos de Personas            | Los administradores pueden modificar la información de las personas cuando sea necesario.                                    |
```
Comment (AntonioRodriguezRuiz) [inline docs/Información del servicio/Casos de uso.mdx:15]: This is more like "patients" rather than people

**Review by AntonioRodriguezRuiz — CHANGES_REQUESTED** [review]
I liked it a lot, I think its a great way to have everything under control. I would like to see however an embed or link to the Product Backlog. That way we can also see the current state of everything

**Comment by alvarobernal2412** [comment]
Done!! I hope my commit has met your requirements😊

**Review by AntonioRodriguezRuiz — APPROVED** [review]
Seems Good

**Pull request merged**